include/multi_blas_helper.cuh

```diff
@@ -121,32 +194,32 @@ namespace quda
        @param[in] scalar_width Width of the scalar that we're
        multiplying by (1 = real, 2 = complex)
     */
-    inline int max_YW_size(int NXZ, QudaPrecision x_prec, QudaPrecision y_prec, bool use_z, bool use_w, int scalar_width, bool reduce, bool multi_1d = false)
+    template <typename Functor>
```

Comment:
update (or temporarily get rid of) doxygen

Reply:
d45340b86